src/frontend/csrs-portal/src/app/app.module.ts

```diff
@@ -88,7 +103,8 @@ export function HttpLoaderFactory(http: HttpClient): TranslateHttpLoader {
         }
       ),
       deps: [OidcSecurityService],
-      multi: false
+      multi: false,
```

Comment:
trailing comma, surprised this is not caught by es-lint

Reply:
removed trailing comma